trade-finance-manager-api/src/v1/controllers/acbs.controller.js

```diff
@@ -22,7 +22,7 @@ const addToACBSLog = async ({
       status: 'Running',
       instanceId: acbsTaskLinks.id,
       acbsTaskLinks,
-      submittedDate: moment().format(),
+      submittedDate: formatISO(new Date()) // This has slightly different behaviour but shouldn't cause any breaking changes
```

Comment:
Unsure on whether to leave like this or hand roll a helper to format them exactly the same
```
moment().format() // 2024-03-01T13:52:20+00:00
formatISO(new Date()) // 2024-03-01T13:52:20Z
```

Reply:
It's tempting to create a mapper to convert `Z` into `+00:00` but I'd need to spend some time testing to make sure each behave the same in BST

Reply:
I think we've talked about this but just in case -- I'm happy with whatever, I'd lean slightly towards a +00:00 helper but that might be overkill -- might be worth checking with Abhi